docs/use-cases/exchange.md

```diff
@@ -0,0 +1,136 @@
+
+Otherwise, you can create a new public/private key pair using [`web3.eth.accounts.create`](https://web3js.readthedocs.io/en/v1.2.0/web3-eth-accounts.html#create) function provided by the web3 library. 
+
+> This document doesn't cover best practices for managing and storing keys due to a variety of methods different exchanges use to achieve a high level of security guarantees on their platform.
+
+#### 2.1.2 Query Blockchain
+
+It's important to understand how to query data from the OMG Network. There are several things that might be important for any exchange:
+- Transaction details - use `getTransaction(id)` function by the [`omg-js`](https://docs.omg.network/omg-js/)
+- Block details - use `getTransactions(filters)` function by the [`omg-js`](https://docs.omg.network/omg-js/)
+- Address balance - use `getBalance(address)` function by the [`omg-js`](https://docs.omg.network/omg-js/). For more details, check [`Retrieve Balances`](https://docs.omg.network/network/balances) guide
+- Network fees - use `getFees` function by the [`omg-js`](https://docs.omg.network/omg-js/)
+
+You can find more details about these functions using the following resources:
+- [omg-js library](https://github.com/omgnetwork/omg-js) - the official JavaScript library that allows to implement the required functionality
+- [omg-js documentation](https://docs.omg.network/omg-js/) - a list of all functions of the omg-js library
+- [omg-js tutorials](https://docs.omg.network/network/start) - installation process, code samples, and various tips for using the omg-js library
+- [omg-js-samples](https://github.com/omgnetwork/omg-js-samples) - a JavaScript project that contains implementation for the core features of the OMG Network
+- [blockchain explorer](https://docs.omg.network/environments) - a tool that allows to view transactions and blocks that happen on the network
+
+#### 2.1.3 Make Deposits
+
+The `Deposit` terminology for an exchange has a different meaning from a deposit on the OMG Network. It's very important to distinguish between those two:
+1. OMG Network deposit - represents an ETH transaction to the corresponding `Vault` smart contract on the Ethereum network. This creates a new deposit on the OMG Network and allows the use of these funds until a user decides to withdraw them back to Ethereum.
+2. Exchange deposit - represents a standard transaction on the OMG Network. A safe number of confirmations for a given exchange should equal to a safe number of confirmations on the Ethereum network. This is known as [`deposit finality period`](https://docs.omg.network/glossary#deposit-finality-period) and is currently set to 10 blocks. The confirmations are counted on the Ethereum because the OMG Network relies on rootchain (Ethereum) security and creates blocks only when new transactions are being formed (i.e. on-demand). 
```

Comment:
Is 'Exchange deposit' the right term? It seems what we're actually describing is a standard L2 transaction.

Reply:
I'm not sure there's an established term for this. I just use the term that most of the exchanges have. They have deposits, trades, withdrawals, thus Exchange deposit. The description explains that it's just a simple transaction on the OMG Network.